.github/workflows/go.yml

```
@@ -0,0 +1,132 @@
+        run: go test -race -shuffle=on -count=1 -covermode=atomic -coverpkg=./... -coverprofile=coverage.out ./...
+
+      # step 5: upload coverage
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v5
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+
+  benchmark:
+    name: Benchmark
+    runs-on: ubuntu-latest
+    if: false
```

actionlint (1.7.10)

[error] constant expression "false" in condition. remove the if: section

(if-cond)